pdr_backend/util/web3_accounts.py

```diff
@@ -0,0 +1,98 @@
+import sys
+from os import getenv
+from typing import List
+from enforce_typing import enforce_types
+
+from eth_account import Account
+from pdr_backend.ppss.web3_pp import Web3PP
+from pdr_backend.contract.token import NativeToken, Token
+from pdr_backend.util.contract import get_address
+from pdr_backend.util.mathutil import to_wei, from_wei
+
+
+@enforce_types
+def create_accounts(n_accounts: int):
+    """
+    @description
+        Create new accounts using the web3 provider and print the private_key and the public_key
+    """
+    # loop through n_wallets
+    for i in range(n_accounts):
+        # pylint: disable=no-value-for-parameter
+        new_account = Account.create()
+        print(f"\nWallet #{i}")
+        print(f"Private Key: {new_account._private_key.hex()}")
+        print(f"Public Key: {new_account.address}")
+
+
+@enforce_types
+def _get_account_balances(checksum_address: str, web3_pp: Web3PP):
+    """
+    @description
+        Get account balances from an address
+    """
+    # loop through n_wallets
+    native_token = NativeToken(web3_pp)
+    OCEAN_addr = get_address(web3_pp, "Ocean")
+    OCEAN_token = Token(web3_pp, OCEAN_addr)
+
+    native_token_balance = native_token.balanceOf(checksum_address)
+    OCEAN_balance = OCEAN_token.balanceOf(checksum_address)
+
+    return native_token_balance, OCEAN_balance
+
+
+@enforce_types
+def view_accounts(addresses: List[str], web3_pp: Web3PP):
+    """
+    @description
+        View account balances for multiple addresses
+    """
+    # loop through n_wallets
+    for address in addresses:
+        checksum_address = web3_pp.web3_config.w3.to_checksum_address(address.lower())
```

Comment:
I would also add this check in the parse arguments part. By this point, the addresses should be definitely checksummable, or preferably already converted (checksummed).

Reply:
this would also remove the need for _get_account_balances as a separate function. Another advantage is that you only need to get the OCEAN_addr once (before the loop), while now you retrieve it for *each* address

Reply:
```
I would also add this check in the parse arguments part. By this point, the addresses should be definitely checksummable, or preferably already converted (checksummed).
```

The current to_checksum_address requires PPSS object which isn't instantiated in the argparse/cli_arguments module at the validation phase.

We can do this by importing `eth_utils.to_checksum_address` inside cli_arguments.py and just validate it directly.

I agree that it's easier to checksum at the entry points and let logic cascade down, but I'm not sure this has been addressed at every entrypoint. I'll update this.

Reply:
```
this would also remove the need for _get_account_balances as a separate function. Another advantage is that you only need to get the OCEAN_addr once (before the loop), while now you retrieve it for each address
```

The checksum() and get_balances() calls are unrelated. I'll modify this a bit so there are less redundant calls.

Reply:
Added checksum and cleaned up calls inside `web3_accounts.py`